### PR TITLE
Review PR for repo-readme.md

### PR DESCRIPTION
**Pull request by dominicbetts**
# Guidelines for reviewers
Using this Pull Request (PR), you can:

- Add general comments about the article in this PR
- Add a comment to a specific line in one of the files that make up this article
- Propose edits to the article

You can view the published version of this article at:
https://azure.microsoft.com/documentation/articles/repo-readme/

## Add general comments
To add a general comment, use this page (**Conversation** tab)

## Add comments to a specific line
To add a comment to specific line, go to the **Files changed** tab, hover your mouse pointer over the line in the file and click the blue **+** to add a new line comment.

## Propose edits to the article
To propose edits to this article, go to the **Files changed** tab, find the file you want to edit and click the pencil icon (**Change this file using the online editor**), and make your changes. When you are done you will be prompted to create a PR that contains your proposed changes.

> Note: The PR with your proposed changes will be pushing changes to the dominicbetts/azure-content-pr:edit-repo-readme branch.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- [Azure IoT device SDK for C](c/readme.md)
+- [Azure IoT device SDK for Node.js](node/device/core/readme.md)
+- [Azure IoT device SDK for Java](java/device/readme.md)
+- [Azure IoT device SDK for .NET](csharp/device/readme.md)
+
+Each language SDK includes sample code and documentation in addition to the library code.
+
+### OS platforms and hardware compatibility
+
+Azure IoT device SDKs can be used with a broad range of OS platforms and devices. See [OS Platforms and hardware compatibility](https://azure.microsoft.com/en-us/documentation/articles/iot-hub-tested-configurations/).
```
Comment (sethmanheim) [inline readme.md]: Remove locale from URL.

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+The Azure IoT Service SDKs help you to build applications that interact with your devices and manage device identities in your IoT hub.
+
+- [Azure IoT service SDK for .Net](csharp/service/README.md)
+- [Azure IoT service SDK for Node.js](node/service/README.md)
+- [Azure IoT service SDK for Java](java/service/readme.md)
+
+## Contribution, feedback and issues
+
+If you would like to become an active contributor to this project please follow the instructions provided in the [contribution guideline](contribute.md).
```
Comment (sethmanheim) [inline readme.md]: guidelines (make plural)

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+If you encounter any bugs or have suggestions for new features, please file an issue in the [Issues](https://github.com/Azure/azure-iot-sdks/issues) section of the project.
+
+## Support
+
+If you are having issues using one of the packages or using the Azure IoT Hub service that go beyond simple bug fixes or help requests that would be dealt within the issues section of this project, the Microsoft Customer Support team will try and help out on a best effort basis.
+To engage Microsoft support, you can create a support ticket directly from the [Azure portal](https://ms.portal.azure.com/#blade/Microsoft_Azure_Support/HelpAndSupportBlade).
+Escalated support requests for Azure IoT Hub SDKs development questions will only be available Monday thru Friday during normal coverage hours of 6 a.m. to 6 p.m. PST.
+Here is what you can expect the Microsoft Support to be able to help with:
```
Comment (sethmanheim) [inline readme.md]: Delete "the"

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+This folder contains various build scripts to build the libraries.
+
+### /doc
+
+This folder contains the following documents that are relevant to all the language SDKs:
+
+- [Set up IoT Hub](doc/setup_iothub.md) describes how to configure your Azure IoT Hub service.
+- [Manage IoT Hub](doc/manage_iot_hub.md) describes how to provision devices in your Azure IoT Hub service.
+- [FAQ](doc/faq.md) contains frequently asked questions about the SDKs and libraries.
+- [OS Platforms and hardware compatibility](https://azure.microsoft.com/en-us/documentation/articles/iot-hub-tested-configurations/) describes the SDK compatibility with different OS platforms as well as specific device configurations.
```
Comment (sethmanheim) [inline readme.md]: Same comment as above, must remove en-us from the URL.

File: readme.md (hunk 1 above)
Comment (dominicbetts) [inline readme.md]: Done

File: readme.md (hunk 2 above)
Comment (dominicbetts) [inline readme.md]: Done

File: readme.md (hunk 3 above)
Comment (dominicbetts) [inline readme.md]: Done

File: readme.md (hunk 4 above)
Comment (dominicbetts) [inline readme.md]: Done

**Comment by dominicbetts** [comment]
Processed all feedback up to here

**Pull request closed without merging**